packages/grafana-e2e/src/flows/configurePanel.ts

```diff
@@ -90,7 +90,13 @@ export const configurePanel = (config: PartialAddPanelConfig | PartialEditPanelC
       e2e.components.Panels.Panel.title(panelTitle).click();
       e2e.components.Panels.Panel.headerItems('Edit').click();
     } else {
-      e2e.components.PageToolbar.item('Add panel').click();
+      try {
+        // Depending on the screen size, the "Add panel" button might be hidden
+        e2e.components.PageToolbar.item('Show more items').click();
+        e2e.components.PageToolbar.item('Add panel').last().click();
+      } catch (e) {
+        e2e.components.PageToolbar.item('Add panel').click();
+      }
```

Comment:
for the record, simply doing `e2e.components.PageToolbar.item('Add panel').click({force: true});` also works because the the button is actually there but hidden (but I guess it's better to click in the visible button).